### PR TITLE
[DI2-340] Show a warning to users on older versions of Firefox

### PR DESCRIPTION
**Pull request by jonkoops**

### CODE REVIEW

File: src/app/AppBody.tsx [hunk 1]
```diff
@@ -85,6 +86,7 @@ const AppBody: FunctionComponent<AppBodyProps> = ({
         <>
           <AppContainer id={APP_CONTAINER_ID} className="main-container">
             <NotificationAlert />
+            <BrowserSupportAlert />
```
Comment (janjaap) [inline src/app/AppBody.tsx:89]: Why would you show an alert for something that people do not have control over?

**Review by chrisvanmook — APPROVED** [review]

File: src/app/AppBody.tsx (hunk 1 above)
Comment (alimpens) [reply src/app/AppBody.tsx:89]: I think an alert is a nice middle ground. Dropping support is a bit much, but showing an alert might motivate people to call ICT, which might help in solving the core issue.

File: src/app/components/BrowserSupportAlert/BrowserSupportAlert.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  // Make sure to update this with the correct version according to the release schedule:
+  // https://wiki.mozilla.org/Release_Management/Calendar
+  if (!isFirefox || version.major >= 78) {
+    return null
+  }
+
+  return (
+    <Alert level="warning" dismissible>
+      Uw browser is verouderd en zal binnenkort niet langer ondersteund worden. Medewerker bij de
+      Gemeente? Zorg dan dat u gebruik maakt van ADW Next.
```
Comment (alimpens) [inline src/app/components/BrowserSupportAlert/BrowserSupportAlert.tsx:27]: Maybe we should rethink this message. People using TGW's can't use ADW Next.

Comment (jonkoops) [reply src/app/components/BrowserSupportAlert/BrowserSupportAlert.tsx:27]: Yeah, waiting for feedback on this one.

File: src/app/AppBody.tsx (hunk 1 above)
Comment (jonkoops) [reply src/app/AppBody.tsx:89]: They have control to pressure IT, that's what we want. They won't listen to us, but I bet they will listen to 3000 angry users.

File: src/app/components/BrowserSupportAlert/BrowserSupportAlert.tsx (hunk 2 above)
Comment (jonkoops) [reply src/app/components/BrowserSupportAlert/BrowserSupportAlert.tsx:27]: Got approval this is good to go.

**Pull request merged**